data/hai/unfettered side missions.txt

```diff
@@ -0,0 +1,531 @@
+			`	Their tone suddenly changes as they raise their voice for everyone to hear, "I challenge you! The winner will earn the right to own this honorable relic!"`
+			choice
+				`	"What do I get if I win?"`
+					goto win
+				`	"Can't I just sell it to you?"`
+			`	"No, because in the eyes of my brethren that would mean I, too, did not deserve this ship. They would then challenge me for it."`
+			label win
+			`	"You will win the right to access our worlds for just a meager daily fee of 500 credits. In order to keep our respect, this Ancient Ladybug may not be destroyed or sold. This is a priceless artifact. You will not be allowed to land so easily as you just did, should you refuse to prove your worth.`
+			`	"Beware, for there are no rules, other than the fact you may only use this one ship. Tell me then, what say you? Will you take to the skies, or are you too afraid?"`
+			choice
+				`	"I accept."`
+				`	"I have nothing to prove to you."`
+					decline
+			`	"Good! I will of course not be fighting you personally, as it is too risky to do so. I would also not want to deny my brethren the opportunity to prove themselves against a mighty human warrior!"`
+			`	You are not sure if they are being serious or not, but their friends seem to be in for it. Their leader approaches you. "So you think you deserve to fly this mighty ship of the Hai empire?"`
+			`	"Yeah, what makes you think that, human?" Another one adds.`
+			choice
+				`	(Say nothing.)`
+				`	"Are you trying to scare me out of the fight?"`
+					goto scare
+			`	"I guess we'll find out." Says a third.`
+			label scare
+			`	The leader gets closer and addresses you in a more serious tone, "You should be aware, <last>, that this is not a friendly competition between allies. You should see it more as a fight for you to truly earn your place with this ship, but as far as I am concerned, you are an enemy. Make no mistakes about that, this fight has only two rules: nobody other than us here may interfere, and being disabled results in losing the fight. Do not even think about tricking us by switching to something other than this Ladybug, or we will not bother showing up! You must prove your worth for this prize ship in this ship and with only this ship."`
+			`	"And do not even think about capturing our ships. Errors happen, so we can understand some may die in glorious battle, but otherwise looting us is as far as we will allow!"`
+			# This is a "warning" for the incoming extra ship that happens because you killed/captured 10 Shield Beetles
+			# (But in reality with disabling a lot of ships it goes faster than that)
+			`	They suddenly look at you intensively, straight in the eyes, "We sometimes bend the rules for our friends, but that is not your case," they say with a sudden change of tone, before leaving.`
```

Comment:
Can you elaborate on why this exists? I don't quite understand the purpose of this line, even with the explanation above.

Reply:
they're angry about you bc you killed their kind and they can't hide it

with the PR to make it so disabling is ignored this means bad stuff happened

Reply:
Okay, got it. I'm still struggling with this line - I don't think I like the current wording, but I'm not sure how to change it just yet either. I'll get back to you on this.